upgrade/scripts/upgrade_python_package.py

```diff
@@ -216,7 +222,13 @@ def is_cloudsmith_url_valid(cloudsmith_url):
 
 def is_package_already_installed(package):
     results = pip("list", "--format", "json")
-    parsed_results = json.loads(results)
+    try:
+        decoder = json.JSONDecoder()
+        parsed_results, index = decoder.raw_decode(results)
+    except json.JSONDecodeError as e:
+        msg = f"Error occurred while decoding pip list at ${index} index."
+        logging.error(msg)
+        raise PipFormatDecodeFailed(msg)
```

Comment:
We could catch this exception in `install_wheel` and print a message saying that it might be necessary to upgrade pip